src/app/pages/registration/services/registration-form-configuration/registration-form-configuration.service.ts

```diff
@@ -107,11 +107,11 @@ export class RegistrationFormConfigurationService {
               required: true,
             },
             validators: {
-              validation: [Validators.pattern('true')],
+              validation: [Validators.requiredTrue],
```

Comment:
pls do it the same way as in the checkout-review.component  
{
        type: 'ish-checkout-review-tac-field',
        key: 'termsAndConditions',
        props: {
          required: true,
          validation: {
            show: true,
          },
        },
        validators: {
          validation: [Validators.pattern('true')],
        },
        validation: {
          messages: {
            pattern: 'checkout.tac.error.tip',
          },
        },
      },